test/controllers/declaration/CommodityMeasureControllerSpec.scala

```diff
@@ -73,56 +71,68 @@ class CommodityMeasureControllerSpec extends ControllerSpec {
 
   "Commodity Measure controller" should {
 
-    "return 200 (OK)" when {
-
-      "display page method is invoked and commodity measure cache is empty" in {
-
-        withNewCaching(aDeclaration())
-        val result = controller.displayPage(Mode.Normal, "itemId")(getRequest())
+    "return 200 (OK) on displayPage" when {
+      onJourney(STANDARD, SUPPLEMENTARY, CLEARANCE) { request =>
```

Comment:
Should we have a test to check the other two journey type actions (redirect to choice page)?